draft-ietf-rtcweb-security-arch.xml

```diff
@@ -1550,14 +1421,13 @@ extension-att-value = 1*(%x01-09 / %x0b-0c / %x0e-3a / %x3c-ff)
   https://example.com/.well-known/idp-proxy/example
   ]]></artwork>
               </figure>
-<!--  [[TODO: suggested by mt]]
-
-
               <t>
-                The IdP proxy MUST be prevented from redirecting or navigating
-                the IdP frame to another site.
+                The IdP MAY redirect requests to this URL, but they MUST retain
+                the "https" scheme.  This changes the effective origin of the
+                IdP, but not the domain of the identities that the IdP is
+                permitted to assert and validate.
```

Comment:
This seems like it could use another sentence. E.g.,

"For instance, if example.com redirects requests to example.org, the identity MUST still be regarded as scoped by example.com". Is that what you mean?